### PR TITLE
Guide: Try it with Postman

### PR DESCRIPTION
**Pull request by philsturgeon**

### CODE REVIEW

File: src/_guides/bump-sh-tutorials/2024-04-28-try-requests-in-postman.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+![](/images/guides/try-it-out/postman-new-api.png)
+
+Then you can configure which Git repository your API should be connected to.
+
+![](/images/guides/try-it-out/postman-connect-repository.png)
+
+Next, point Postman to the "API schema file", which is their way of saying "API description document." Whatever you call it, this is the same `openapi.yaml` or similar document that you use to deploy to Bump.sh.
+
+It's possibly sat in the root of your repository, but if you're not sure, you can look at yor `.github/workflows/bump.yml` and see where the deploy command is looking for the `file: {filename}`.
```
Comment (ChristopheDujarric) [inline src/_guides/bump-sh-tutorials/2024-04-28-try-requests-in-postman.md:37]:
```suggestion
It's possibly sat in the root of your repository, but if you're not sure, you can look at your `.github/workflows/bump.yml` and see where the deploy command is looking for the `file: {filename}`. 
```

**Review by ChristopheDujarric — CHANGES_REQUESTED** [review]
Very cool, thanks @philsturgeon !

Only one small typo to fix. Can you also please update the filename to `2024-05-13-try-requests-in-postman.md` to better reflect the latest update?

**Comment by philsturgeon** [comment]
@ChristopheDujarric good spot! I've done that and also added a bunch of Alt tags to the images.

**Comment by ChristopheDujarric** [comment]
Thanks @philsturgeon !

A helpful resource, for sure!

**Pull request merged**